src/main/java/edu/cornell/kfs/sys/CUKFSParameterKeyConstants.java

```diff
@@ -52,5 +52,6 @@
         public static final String DEFAULT_NAME_SPACE_CODE = "KFS-SYS";
         public static final String DEFAULT_COMPONENT = "PurgeTablesStep";
         public static final String DEFAULT_PARAMETER_NAME = "DEFAULT_NUMBER_OF_DAYS_OLD";
+        public static final String PS_POSITION_DATA_WD_PARAMETER_NAME = "PS_POSITION_DATA_WD_NUMBER_OF_DAYS_OLD";

```

Comment:
Is this constant needed?

Reply:
Isn't it our normal practice to have a constant defined so that we do not need to hard code a parameter name as a string where we use it in the code?
Based on your other noted concerns, I will change the constant to PS_POSITION_DATA_WD_NUMBER_OF_DAYS_OLD. so it matches the parameter name.

Reply:
Hi Nancy, yes, normal practice is to have a constant for parameter names. But this particular one is not used anywhere in the code so I am not sure it is needed.